src/override.js

```diff
@@ -126,11 +126,16 @@ export default function (Vue) {
    *
    * @param {Store} store
    * @param {Function} action
+   * @param {String} key
    */
 
-  function makeBoundAction (store, action) {
+  function makeBoundAction (store, action, key) {
     return function vuexBoundAction (...args) {
```

Comment:
actually, we should do the `typeof action` check and warn in `makeBoundAction` before returning the bound action. This way the user gets the warning right at instance creation time instead of when they actually call the action.